### DISCUSSION TITLE
Nuxt content - code highlight not working with tailwind typography plugin

### DISCUSSION THREAD

**Discussion by pedrodruviaro** [nuxt/nuxt#25293] [Questions]
I am creating a small blog with Nuxt Content and Tailwind. My ContentDoc is rendered like this:

```
    <div
      class="prose-sm md:prose dark:prose-invert md:dark:prose-invert md:max-w-[50rem] mx-auto"
    >
      <ContentDoc>
        <template #not-found>
          <div>
            <h2>Você se perdeu, essa página não existe... :(</h2>
            <NuxtLink to="/">Voltar</NuxtLink>
          </div>
        </template>
      </ContentDoc>
    </div>
```

My Nuxt config file:

```
export default defineNuxtConfig({
  devtools: { enabled: true },
  modules: ["@nuxtjs/tailwindcss", "@nuxt/content", "@nuxtjs/color-mode"],
  content: {
    highlight: {
      theme: {
        default: "dracula",
        dark: "dracula",
      },
    },
  },
  colorMode: { classSuffix: "", preference: "system", fallback: "dark" },
  nitro: {
    prerender: {
      routes: ["/sitemap.xml"],
    },
  },
  app: {
    pageTransition: { name: "page", mode: "out-in" },
  },
  routeRules: {
    "/": { prerender: true },
  },
})
```

I've tried various ways to make the code appear with the correct theme, but nothing has worked. Any idea what might be happening?

**Comment by TheAlexLichter** [comment]
Hey! Could you kindly provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

**Reply by pedrodruviaro** [reply]
My code: 
![image](https://github.com/nuxt/nuxt/assets/82953655/4ef30774-d368-4d5b-9ddb-b629384f89b1)

Stackblitz:
![image](https://github.com/nuxt/nuxt/assets/82953655/5b63ea0b-266a-4ee6-9f83-65963dbe7fd5)

I uploaded the code to StackBlitz, and it worked. I noticed that the HTML structure in my code is different (images attached).

**Reply by pedrodruviaro** [reply]
Update: I updated the packages, and the theme appeared in production. The bug persists only in the local environment. I'm already happy with that.

**Reply by TheAlexLichter** [reply]
Can you check that you are on 3.9.3? There was a CSS fix for dev in the latest patch release.

**Reply by pedrodruviaro** [reply]
Here's my dependencies:

```
  "devDependencies": {
    "@nuxt/devtools": "latest",
    "@nuxtjs/color-mode": "^3.3.2",
    "@nuxtjs/tailwindcss": "^6.10.4",
    "@tailwindcss/typography": "^0.5.10",
    "nuxt": "^3.9.3",
    "vue": "^3.4.15",
    "vue-router": "^4.2.5"
  },
  "dependencies": {
    "@nuxt/content": "^2.10.0",
    "sitemap": "^7.1.1"
  }
```
